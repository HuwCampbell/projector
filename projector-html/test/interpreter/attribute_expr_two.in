<a { [(Attribute (AttributeKey "marquee") (AttributeValue "true")), (Attribute (AttributeKey "foo") (AttributeValue "bar"))] }/>