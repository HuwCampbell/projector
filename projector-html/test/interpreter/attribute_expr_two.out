VoidElement
  "a" [ Attribute "marquee" "true" , Attribute "foo" "bar" ]